staging/src/k8s.io/cloud-provider/sample/basic_main.go

```diff
@@ -62,8 +60,10 @@ func main() {
 		os.Exit(1)
 	}
 
+	cloudConfigFile := c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile
+
 	// initialize cloud provider with the cloud provider name and config file provided
-	cloud, err := cloudprovider.InitCloudProvider(sampleCloudProviderName, sampleCloudProviderConfigFile)
+	cloud, err := cloudprovider.InitCloudProvider(sampleCloudProviderName, cloudConfigFile)
```

Comment:
nit: I would just inline this:
```
cloudprovider.InitCloudProvider(sampleCloudProviderName, c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile)
```

Reply:
sure, will update

Reply:
It initially was trying to show the cloudConfigFile path could be modified by cloud provider if it is needed. Sorry for the confusion. Would you mind to update `https://github.com/kubernetes/kubernetes/blob/4efd56485d10058d69b9f1cd7eb67470c33f12cf/cmd/cloud-controller-manager/main.go#L78` as well? It might be misleading as well. Thank you

Reply:
sure, will do!